polaris-react/config/rollup/plugin-styles.js

```diff
@@ -148,16 +148,9 @@ module.exports.styles = function styles({
 
       let sassOutput;
       try {
-        sassOutput = nodeSass
-          .renderSync({
-            data: source,
-            file: id,
-            outputStyle: 'compact',
-            includePaths: [path.dirname(id)],
-          })
-          .css.toString();
+        sassOutput = (await sass.compileAsync(id, {style: 'compressed'})).css;
```

Comment:
We currently use the expanded style (i.e. https://unpkg.com/browse/@shopify/polaris@10.4.1/build/esm/styles.css)

I reckon we should stick to that.

Reply:
Good call! I assumed the equivalent of `compact` was `compressed`. Updated: https://github.com/Shopify/polaris/pull/7321/commits/3467ae2fca069858c96614694ec826a5135bfc87

Reply:
oh duh, I'm wrong and skim-reading sorry.

In node-sass there were 4 output styles, in dart-sass there's only two. `compressed` and `expanded` continue to exist, while `compact` and and `nested` were removed ([ooold context on what those look like](https://inchoo.net/dev-talk/tools/sass-output-styles/)).


I'd lean towards the more human-readable output when we publish to npm so it is easier to identify build output differences over time, so I'd prefer `expanded` over `compressed`.